### DISCUSSION TITLE
✅ Add SVG support to fontawesome

### DISCUSSION THREAD

**Discussion by ClausR1990** [elementor/elementor#13620] [Feature Request]
The Font Awesome features SVG instead of font format. This improve load speed.
But right now, sizing and color of for the icon when using SVG mode is not support in Elementor Editor.

Example Elementor editor uses FILL Property, but should use COLOR.

**Comment by nicholaszein** [comment]
Hello,

this feature was introduced with the release of the **Inline Font Icons** experimental feature in version 3.4.0 in August 2021.

Since version 3.18.0 this feature is active by default on new sites, and since version 3.20.0 it is active by default on all installations.

Kind regards
